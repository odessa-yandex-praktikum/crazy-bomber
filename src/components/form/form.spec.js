import React from 'react';
import renderer from 'react-test-renderer';
import {Form} from './index';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <Form
                classForm={`form__theme-${theme}`}
                arrayInputs={[<input />]}
                arrayButtons={[<button />, <button />]}
            />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
