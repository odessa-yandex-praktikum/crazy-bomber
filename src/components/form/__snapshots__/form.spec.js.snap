// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form signin__form"
>
  <input />
  <div
    className="form__buttons-container"
  >
    <button />
    <button />
  </div>
</div>
`;
